### PR TITLE
Trainings for champions program

### PR DESCRIPTION
**Pull request by yabellini**
Blog post wrapping up the training stage of the champions program.

### CODE REVIEW

File: content/blog/2023-06-16-champions-program-training/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: "rOpenSci Champions Program Teams: Meet César and Marc"
```
Comment (yabellini) [inline content/blog/2023-06-16-champions-program-training/index.md:2]:
```suggestion
title: "Training at the rOpenSci Champions Program Pilot"
```

File: content/blog/2023-06-16-champions-program-training/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+This workshop is a 60-minute walkthrough of how to contribute to base R. The outline includes How we can contribute to a bug, confirm a bug, diagnose a bug, and propose a fix to code/documentation.  
+
+The workshop shows tools and strategies to find a bug to work on and what features make it good for first-timer to work on. It has examples and exercises on how to triage bugs.  Then show how to debug code/documentation with collaborative hand on. Next, show how to fix a bug by commenting on Bugzilla and a demo on creating a patch with git workflow. Finally, share pointers to further help and resources.
+
+This training was developed for the Champion Program, taught by [Hether Turner](/author/heather-turner/), and is open to all champions and mentors.
+
+Training material: https://ropensci.org/events/2023-05-18-how-to-contribute-base-r/ 
+
+## Trainning for the Mentors
+
```
Comment (yabellini) [inline content/blog/2023-06-16-champions-program-training/index.md:80]:
```suggestion
Mentors gain mentoring skills during this training and orientation (active listening, effective questioning, and giving feedback). They receive resources for their mentoring roles, such as the Mentors Guidelines, Meeting Templates, and details of the logistics of the champions program.

This training was developed for the Champion Program, taught by [Yani Bellini Saibene](/author/yanina-bellini-saibene/), and all mentors must complete it. 

Training material: https://ropensci-training.github.io/ropensci-mentors/
```

**Comment by yabellini** [comment]
Hi @maelle, @maurolepore, @paocorrales, and @hturner, this is a blog post about the training on the champion program.  I added you as authors because you delivered some of the sessions.  Please, review the text and suggests improvements.  The publication date is July 21, I hope you can review before that date.

Thanks so much!

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+With a new cohort starting this year we plan to improve our schedulle of each training session allowing participat to choose between two options of date taking into account time zones, and recording the material to allow them to watch after the meeting. We will develop guidelines for increase the accesibility of each session and we will change the order od the workshops. 
+
+With this actions we intend to address some of the dificulties related to time zone, accesibility and participants' commitments in their jobs or studies. 
+
+
+## Achivements
+
+We developed a new curriculum involving not only rOpenSci staff but also champions and mentors and other communities, contributing to the grow of the connection of our community. 
+
+We also published all the material with open licenses, that allow other organization, like the The Netherlands eScience Center. adapt and reuse our material for trainig mentors for their fellow program.
```
Comment (maurolepore) [inline content/blog/2023-07-21-champions-program-training/index.md:130]:
```suggestion
We also published all the material with open licenses, that allow other organization, like the The Netherlands eScience Center adapt and reuse our material for trainig mentors for their fellow program.  
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+At the end of each workshop, we asked those who participated to fill out an anonymous survey telling us three things: 
+
+* What did you like the most?
+
+* What did you like the least? 
+
+* Can you rate this meeting? (in a scale from 1-worst to 5-best). 
+
+We use this feedback to improve the next sesion of the training. The result show the average score for all the training is **4.6**. which let us know that the attendees are satisfied with the workshops.
```
Comment (maurolepore) [inline content/blog/2023-07-21-champions-program-training/index.md:102]: Here I see two typos:

```suggestion
We use this feedback to improve the next session of the training. The results show the average score for all the training is **4.6**. which let us know that the attendees are satisfied with the workshops. 
```

**Review by maurolepore — APPROVED** [review]
Thanks a lot @yabellini!

I have only minor suggestions:

* I reviewed the structure and I love it -- it seems complete and clear.
* I reviewed the section that describes the workshop I lead and did minor suggestions -- mostly rearranging text to be less repetitive. Feel free to disregard.
* I skimmed the rest and noticed a few typos.

**Comment by yabellini** [comment]
Thanks @maurolepore ! I already apply your changes :-)

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+In addition mentors are invited to all the training offered to champions as an optional activity.
+
+## Training for non-selected applicants
+
+We also conduct workshops for those not selected to be champions. The aim of this workshops is to provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.
+
+### [Developing Software Together](https://paocorrales.github.io/git-developing-software-together/)  
+
+It is a three-hour course on using Git and GitHub to collaboratively develop open-source software with the R language. The workshop presents how to work with Git, using the command line, RStudio, and GitHub. Introduce the concept of version control systems, local and remote repositories, and actions like stage, commit, push, pull, and merge.  Also, show how to collaborate with others through forks and pull requests. The workshop has hands-on and group exercises.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:87]:
```suggestion
It is a three-hour course on using Git and GitHub to collaboratively develop open-source software with the R language. The workshop presents how to work with Git, using the command line, RStudio, and GitHub. It introduces the concept of version control systems, local and remote repositories, and actions like stage, commit, push, pull, and merge.  Also, show how to collaborate with others through forks and pull requests. The workshop has hands-on and group exercises.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+{{< figure src = "feedback_toimprove.jpg" class = "pull-right" alt = "Ranked quantities of five areas of improvement in a bar plot. The aspect are Nothing (12 mentions), Level (5 mentions), More time (4 mentions), Tools (4 mentions), Exercises (3 mentions), and Pace (3 mentions). Each aspect have a comment explaining the specific improvement suggestion or feedback.">}}
+
+The survey data provides us with valuable insights into the participants' perspectives on the strengths and areas for enhancement our trainings and allow us to take action to further enhance future iterations of the training.
+
+
+## Next Step
+
+With a new cohort starting this year we plan to improve our schedulle of each training session allowing participat to choose between two options of date taking into account time zones, and recording the material to allow them to watch after the meeting. We will develop guidelines for increase the accesibility of each session and we will change the order od the workshops.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:121]:
```suggestion
With a new cohort starting this year we plan to improve our schedule of each training session allowing participants to choose between two options of date taking into account time zones, and recording the material to allow them to watch after the meeting. We will develop guidelines to increase the accessibility of each session and change the order of the workshops. 
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+Even when the answers to "What did you like the least?" show that most attendees think there is _Nothing_ they liked the least we identify five areas for improvement: _Level_ of the training, _More time_ for all the content or exercises, _Tools_ used during the workshops,  _Exercises_ during the workshop, _Pace_ of the teaching. In the plot we show the number of mention and each aspect is followed by a comment explaining the specific improvement suggestion or feedback. 
+
+The comments highlight the need for more time to cover content, the desire for better collaboration tools, the request for additional exercises, and the suggestion to offer different levels of the training, for begginers and for competent practictioners. The comments also includes expression of overall satisfaction while mentioning challanges, like time zones.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:112]:
```suggestion
The comments highlight the need for more time to cover content, the desire for better collaboration tools, the request for additional exercises, and the suggestion to offer different levels of training, for beginners and for competent practitioners. The comments also include expressions of overall satisfaction while mentioning challenges, like time zones. 
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+{{< figure src = "feedback_positive.jpg" class = "pull-left" alt = "Ranked quantities of four different positive aspects in a bar plot. The aspect are Material and Resources (15 mentions), Interaction with other participants, (10 mentions), Exercises (8 mentions), and Structure (7 mentions). Each aspect is followed by a comment explaining why it was perceived positively by the participants.">}}
+
+Even when the answers to "What did you like the least?" show that most attendees think there is _Nothing_ they liked the least we identify five areas for improvement: _Level_ of the training, _More time_ for all the content or exercises, _Tools_ used during the workshops,  _Exercises_ during the workshop, _Pace_ of the teaching. In the plot we show the number of mention and each aspect is followed by a comment explaining the specific improvement suggestion or feedback.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:110]:
```suggestion
Even when the answers to "What did you like the least?" show that most attendees think there is _Nothing_ they liked the least we identify five areas for improvement: _Level_ of the training, _More time_ for all the content or exercises, _Tools_ used during the workshops,  _Exercises_ during the workshop, _Pace_ of the teaching. In the plot we show the number of mentions and each aspect is followed by a comment explaining the specific improvement suggestion or feedback. 
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+
+* What did you like the least? 
+
+* Can you rate this meeting? (in a scale from 1-worst to 5-best). 
+
+We use this feedback to improve the next session of the training. The results show the average score for all the training is **4.6**. which let us know that the attendees are satisfied with the workshops. 
+
+When we analyze the responses to "What did you like the most?" in detail we found four positive aspects mentioned: _Interaction with other participants_, _Structure_ of the workshop, _Material and Resources_ shared during the workshop, and _Exercises_ performed during training. 
+
+In the following plot is the detail of the number of mention for each aspect and an example comment explaining why it was perceived positively by the participants. The comments highlight various aspects of the trainings, such as enjoyable interactions, valuable tools provided, opportunity for reflection and a good structure base on practice after instruction.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:106]:
```suggestion
In the following plot is the detail of the number of mentions for each aspect and an example comment explaining why it was perceived positively by the participants. The comments highlight various aspects of the training, such as enjoyable interactions, valuable tools provided, opportunities for reflection and a good structure base on practice after instruction.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+Mentors gain mentoring skills during this training and orientation (active listening, effective questioning, and giving feedback). They receive resources for their mentoring roles, such as the Mentors Guidelines, Meeting Templates, and details of the logistics of the champions program.
+
+This training was developed for the Champion Program, taught by [Yanina Bellini Saibene](/author/yanina-bellini-saibene/), and all mentors must complete it. 
+
+
+In addition mentors are invited to all the training offered to champions as an optional activity.
+
+## Training for non-selected applicants
+
+We also conduct workshops for those not selected to be champions. The aim of this workshops is to provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.
```
Comment (paocorrales) [inline content/blog/2023-07-21-champions-program-training/index.md:83]:
```suggestion
We also conduct workshops for those not selected to be champions. The aim of these workshops is to provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.
```

**Review by paocorrales — COMMENTED** [review]
The blogs (and the plots :wink: ) looks great!

**Comment by yabellini** [comment]
Thank you @paocorrales ! Suggestions applied.

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+{{< figure src = "feedback_toimprove.jpg" class = "pull-right" alt = "Ranked quantities of five areas of improvement in a bar plot. The aspect are Nothing (12 mentions), Level (5 mentions), More time (4 mentions), Tools (4 mentions), Exercises (3 mentions), and Pace (3 mentions). Each aspect have a comment explaining the specific improvement suggestion or feedback.">}}
+
+The survey data provides us with valuable insights into the participants' perspectives on the strengths and areas for enhancement our trainings and allow us to take action to further enhance future iterations of the training.
+
+
+## Next Step
+
+With a new cohort starting this year we plan to improve our schedule of each training session allowing participants to choose between two options of date taking into account time zones, and recording the material to allow them to watch after the meeting. We will develop guidelines to increase the accessibility of each session and change the order of the workshops. 
+
+With this actions we intend to address some of the dificulties related to time zone, accesibility and participants' commitments in their jobs or studies.
```
Comment (hturner) [inline content/blog/2023-07-21-champions-program-training/index.md:123]: could be family or other commitments
```suggestion
With these actions we intend to address some of the difficulties related to time zones, accessibility and participants' other commitments. 
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+  - Paola Corrales
+date: '2023-07-21'
+tags:
+  - community
+  - champions program
+  - training
+description: ""
+---
+
+Our first cohort of the [rOpenSci Champions Program](/champions/) finished the first phase of their training. In this article, we share the curriculum created for the champions, mentors, and applicants and what we learned in this pilot.
```
Comment (maelle) [inline content/blog/2023-07-21-champions-program-training/index.md:19]: add a note that you can apply to the second cohort now?

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+participants explore designing and facilitating engaging and productive virtual meetings and events.
+
+Participants discuss the opportunities and challenges of working together online and outline the “Making a PACT” framework for effective meeting design. In addition to examples of the framework in action, attendees can apply it directly to some example scenarios so that participants feel empowered to use the framework in their own work.
+Participants are welcome to review the CSCCE’s free guidebook on Zenodo ahead of the session: [A guide to using virtual events to facilitation community building: Making a PACT for more engaging virtual meetings and events](https://zenodo.org/record/4987666#.Y-1Y4nbMK5c)
+
+All champions must complete this training.
+
+### [Package Development: The Mechanics](https://rpkgdev-mechanics.netlify.app/)
+
+In this two-hour session with a live demo, a rOpenSci trainer demystifies the creation of an R package. R packages are mostly well-organized folders, with automatic tools to help, such as usethis.
```
Comment (maelle) [inline content/blog/2023-07-21-champions-program-training/index.md:43]:
```suggestion
In this two-hour session with a live demo, an rOpenSci trainer demystifies the creation of an R package. R packages are mostly well-organized folders, with automatic tools to help, such as usethis.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+
+This training was developed for the Champion Program and taught by [Maëlle Salmon](/author/maëlle-salmon/).  It is intended for champions planning to develop a package and have not previously developed or contributed to packages.
+
+
+### [Package Development: Not Rocket Science](https://rpkgdev-rocket.netlify.app/)
+
+This two-hour workshop is for R users who’ve already developed or contributed to packages or have taking the Package Development: The Mechanics workshop. It is intended for champions planning to develop a package or send it to review and want to learn several tips and tricks on package development.
```
Comment (maelle) [inline content/blog/2023-07-21-champions-program-training/index.md:50]:
```suggestion
This two-hour workshop is for R users who’ve already developed or contributed to packages or have taken the "Package Development: The Mechanics" workshop. It is intended for champions planning to develop a package or send it to review and want to learn several tips and tricks on package development.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+## Trainning for the Mentors
+
+### [rOpenSci Champions Program. Mentorship Training and Orientation](https://ropensci-training.github.io/ropensci-mentors/)
+
+Mentors gain mentoring skills during this training and orientation (active listening, effective questioning, and giving feedback). They receive resources for their mentoring roles, such as the Mentors Guidelines, Meeting Templates, and details of the logistics of the champions program.
+
+This training was developed for the Champion Program, taught by [Yanina Bellini Saibene](/author/yanina-bellini-saibene/), and all mentors must complete it. 
+
+
+In addition mentors are invited to all the training offered to champions as an optional activity.
```
Comment (maelle) [inline content/blog/2023-07-21-champions-program-training/index.md:79]:
```suggestion
In addition mentors are invited to all the trainings offered to champions as an optional activity.
```

**Review by maelle — COMMENTED** [review]
Thanks so much! So cool to see all this work that you @yabellini masterfully organized :rocket: 

I made a few comments/suggestions.

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+
+## Training for non-selected applicants
+
+We also conduct workshops for those not selected to be champions. The aim of these workshops is to provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.
```
Comment (yabellini) [inline content/blog/2023-07-21-champions-program-training/index.md:83]:
```suggestion
We also conduct a workshop for those not selected to be champions. The aim of this workshop is to provide an opportunity for all applicants to develop skills in open-source software development for supporting career opportunities as well as future applications to this or other similar programs.
```

File: content/blog/2023-07-21-champions-program-training/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2023-07-21-champions-program-training/index.md:43]: I think we don't usually do that in our blog post.

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 17]
```diff
@@ -0,0 +1,132 @@
...
+  - Paola Corrales
+date: '2023-07-21'
+tags:
+  - community
+  - champions program
+  - training
+description: ""
+---
+
+Our first cohort of the [rOpenSci Champions Program](/champions/) have finished the first phase of their training. In this article, we share the curriculum created for the champions, mentors, and applicants, and what we learned in this pilot.
```
Comment (yabellini) [inline content/blog/2023-07-21-champions-program-training/index.md:19]:
```suggestion
Our first cohort of the [rOpenSci Champions Program](/champions/) has now completed the first phase of their training. In this article, we share the curriculum created for the champions, mentors, and applicants, and what we learned in this pilot.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 18]
```diff
@@ -0,0 +1,132 @@
...
+* technical skills in software development and review.
+
+The final curriculum for this pilot consisted of 5 courses:
+
+### Making a PACT for engaging virtual meetings and events
+
+In this 90-minute mini-workshop hosted by the [Center for Scientific Collaboration and Community Engagement](https://www.cscce.org/),
+participants explore designing and facilitating engaging and productive virtual meetings and events.
+
+Participants discuss the opportunities and challenges of working together online and explore the “Making a PACT” framework for effective meeting design. In addition to examples of the framework in action, attendees apply it directly to some example scenarios, so they feel empowered to use the framework in their own work.
```
Comment (yabellini) [inline content/blog/2023-07-21-champions-program-training/index.md:36]:
```suggestion
Participants discuss the opportunities and challenges of working together online and outline the “Making a PACT” framework for effective meeting design. In addition to reviewing examples of the framework in action, attendees practice applying it  to some example scenarios to empower them to try using  the framework in their own work.
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+Participants are invited to review the CSCCE’s free guidebook on Zenodo ahead of the session: [A guide to using virtual events to facilitation community building: Making a PACT for more engaging virtual meetings and events](https://zenodo.org/record/4987666#.Y-1Y4nbMK5c)
```
Comment (yabellini) [inline content/blog/2023-07-21-champions-program-training/index.md:37]:
```suggestion
Participants are welcome to review the CSCCE’s free guidebook on Zenodo ahead of the session: [A guide to using virtual events to facilitation community building: Making a PACT for more engaging virtual meetings and events](https://zenodo.org/record/4987666#.Y-1Y4nbMK5c)
```

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 20]
```diff
@@ -0,0 +1,132 @@
...
+### [Package Development: The Mechanics](https://rpkgdev-mechanics.netlify.app/)
+
+In this two-hour session with a live demo, an rOpenSci trainer demystifies the creation of an R package. R packages are mostly well-organized folders, with automatic tools to help, such as usethis.
+
+This training was developed for the Champion Program and taught by [Maëlle Salmon](/author/maëlle-salmon/).  It is intended for champions planning to develop a package who have not previously developed or contributed to packages.
+
+
+### [Package Development: Not Rocket Science](https://rpkgdev-rocket.netlify.app/)
+
+This two-hour workshop is for R users who’ve already developed or contributed to packages or have taken the Package Development: The Mechanics workshop. It is intended for champions planning to develop a package or send it to review who want to learn several tips and tricks on package development.
```
Comment (yabellini) [inline content/blog/2023-07-21-champions-program-training/index.md:50]:
```suggestion
This two-hour workshop is for R users who’ve already developed or contributed to packages or have taken the Package Development: The Mechanics workshop. It is intended for champions planning to develop a package or submit it for review and who want to learn several tips and tricks on package development.
```

**Review by yabellini — COMMENTED** [review]
Adding some more edits.

**Comment by yabellini** [comment]
@steffilazerte this is ready for your review. I would like to publish this week.  Perhaps Friday? Thanks!!

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 21]
```diff
@@ -0,0 +1,150 @@
+---
+title: "Training at the rOpenSci Champions Program Pilot"
+author:
```
Comment (steffilazerte) [inline content/blog/2023-07-21-champions-program-training/index.md:3]: I think we need a 'slug' right?
```suggestion
slug: "champions-program-training"
author:
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini 

This is a really lovely wrap-up! It's nice to get an idea of the different workshops and the feedback. I think this will be a great resource to direct potential Champions and Mentors to so they have a sense of what to expect.

I have a lot of suggestions, mostly capitalizing Champions and Mentors, and the Champions Program (you know how picky I am 😉 ), as well as making the descriptions passed tense. I think because the post is about how things went, the descriptions really need to be in the past tense. 

Let me know if I've misunderstood anything, thanks Yani!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-07-21-champions-program-training/index.md [hunk 22]
```diff
@@ -0,0 +1,150 @@
...
+date: '2023-07-21'
+tags:
+  - community
+  - champions program
+  - training
+description: ""
+---
+
+Our first cohort of the [rOpenSci Champions Program](/champions/) has now completed the first phase of their training. In this article, we share the curriculum created for the champions, mentors, and applicants, and what we learned in this pilot.
+
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2023-07-21-champions-program-training/index.md:21]: Excellent idea!

File: content/blog/2023-08-04-champions-program-training/index.md [hunk 23]
```diff
@@ -0,0 +1,154 @@
...
+This training was developed for the Champion Program and taught by [Mauro Lepore](/author/mauro-lepore/). 
+
+
+### [How to contribute to base R](/events/2023-05-18-how-to-contribute-base-r/ )
+
+This workshop was a 60-minute walk-through of how to contribute to base R. It included how to contribute to a bug, confirm a bug, diagnose a bug, and propose a fix to code/documentation.  
+
+The workshop presented tools and strategies to find a bug to work on and discussed which features make a bug good for a first-timer. Using real bug reports, participants practiced identifying good first issues. Then the process of debugging code/documentation was introduced in a collaborative exercise. Participants were briefly shown how to propose a bug fix by commenting on Bugzilla or creating a patch using GitHub. Finally, pointers and resources were shared.
+
+This training was developed and taught by [Heather Turner](/author/heather-turner/), and was open to all Champions and Mentors.
```
Comment (yabellini) [inline content/blog/2023-08-04-champions-program-training/index.md:78]:
```suggestion
This training was developed and taught by [Heather Turner](/author/heather-turner/).
```

File: content/blog/2023-08-04-champions-program-training/index.md [hunk 24]
```diff
@@ -0,0 +1,154 @@
...
+### [rOpenSci Champions Program. Mentorship Training and Orientation](https://ropensci-training.github.io/ropensci-mentors/)
+
+Mentors developed their mentoring skills during this training and orientation, which covered active listening, effective questioning, and giving feedback. They received resources for their mentoring roles, such as the Mentors' Guidelines, Meeting Templates, and details of the logistics of the Champions Program.
+
+This training was mandatory for all Mentors. It was developed for the Champions Program and taught by [Yanina Bellini Saibene](/author/yanina-bellini-saibene/).
+
+
+## Training for non-selected applicants
+
+We also conducted a workshop for those not selected to be Champions. The aim of this workshop was to provide an opportunity for all applicants to develop skills in open-source software development to support their their professional advancement, which may include future applications to this or other similar programs.
```
Comment (yabellini) [inline content/blog/2023-08-04-champions-program-training/index.md:93]:
```suggestion
We also conducted a workshop for those not selected to be Champions. The aim of this workshop was to provide an opportunity for all applicants to develop skills in open-source software development to support their professional advancement, which may include future applications to this or other similar programs.
```

File: content/blog/2023-08-04-champions-program-training/index.md [hunk 25]
```diff
@@ -0,0 +1,154 @@
...
+
+{{< figure src = "feedback_positive.png" class = "pull-left" alt = "Ranked quantities of four different positive aspects in a bar plot. The aspects are Material and Resources (15 mentions), Interaction with other participants, (10 mentions), Exercises (8 mentions), and Structure (7 mentions). Each aspect is followed by a comment explaining why it was perceived positively by the participants.">}}
+
+Even though the answers to "What did you like the least?" show that most attendees think there is _Nothing_ they liked the least, we identify five areas for improvement: _Level_ of the training, _More time_ for all the content or exercises, _Tools_ used during the workshops,  _Exercises_ during the workshop, _Pace_ of the teaching. In the plot we show the number of mentions and each aspect is followed by a comment explaining the specific improvement, suggestion, or feedback.
```
Comment (yabellini) [inline content/blog/2023-08-04-champions-program-training/index.md:120]:
```suggestion
Even though the answers to "What did you like the least?" show that most attendees think there is _Nothing_ they liked the least, we identify five areas for improvement: _Level_ of the training, _More time_ for all the content or exercises, _Tools_ used during the workshops, _Exercises_ during the workshop, and _Pace_ of the teaching. In the plot we show the number of mentions and each aspect is followed by a comment explaining the specific improvement, suggestion, or feedback. 
```

File: content/blog/2023-08-04-champions-program-training/index.md [hunk 26]
```diff
@@ -0,0 +1,154 @@
...
+We also published all the training material with open licenses. This has enabled other organizations, such as [The Netherlands eScience Center](https://www.esciencecenter.nl/), to adapt and reuse our materials for training Mentors for their own [Fellowship program](https://www.esciencecenter.nl/fellowship-programme/).  If you use our training materials, please let us know [by sharing your use case](/usecases/).
+
+Overall, the rOpenSci Champions Program successfully trained participants in essential skills for their role as Champions and Mentors while fostering engagement and collaboration in open-source research software development. 
+
+
+## Acknowledgements
+
+This phase of the Champions Program was made possible by the work and feedback of many people.
+
+We want to thank the CSCCE team, especially _Lou Woodley_, _Camille Santistevan_, and _Maya Sanghvi_, for their support in designing the overall program and for their "Making a PACT" workshop for the Champions.
```
Comment (yabellini) [inline content/blog/2023-08-04-champions-program-training/index.md:148]:
```suggestion
We want to thank the [CSCCE](https://www.cscce.org/) team, especially _Lou Woodley_, _Camille Santistevan_, and _Maya Sanghvi_, for their support in designing the overall program and for their "Making a PACT" workshop for the Champions.
```

File: content/blog/2023-08-04-champions-program-training/index.md [hunk 27]
```diff
@@ -0,0 +1,154 @@
...
+We developed a new curriculum involving not only rOpenSci staff but also Champions, Mentors, and individuals from other communities, which has strengthened connections among rOpenSci and others in our shared ecosystem. 
+
+We also published all the training material with open licenses. This has enabled other organizations, such as [The Netherlands eScience Center](https://www.esciencecenter.nl/), to adapt and reuse our materials for training Mentors for their own [Fellowship program](https://www.esciencecenter.nl/fellowship-programme/).  If you use our training materials, please let us know [by sharing your use case](/usecases/).
+
+Overall, the rOpenSci Champions Program successfully trained participants in essential skills for their role as Champions and Mentors while fostering engagement and collaboration in open-source research software development. 
+
+
+## Acknowledgements
+
+This phase of the Champions Program was made possible by the work and feedback of many people.
```
Comment (yabellini) [inline content/blog/2023-08-04-champions-program-training/index.md:146]:
```suggestion
The rOpenSci Champions Program is funded by [The Chan Zuckerberg Initiative](https://chanzuckerberg.com/). This phase of the Champions Program was made possible by the work and feedback of many people.
```

**Pull request merged**